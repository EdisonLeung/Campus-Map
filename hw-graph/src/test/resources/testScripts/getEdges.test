# Tests the getChildren function by adding edges, printing the edges and then
# adding another edge and printing it to see if new edge is printed

# Create graph
CreateGraph getEdge

# Add Nodes
AddNode getEdge a
AddNode getEdge b

# Add edge
AddEdge getEdge a b AB

# Print children
ListChildren getEdge a

# Add another edge
AddEdge getEdge a a AA

# Print children
ListChildren getEdge a