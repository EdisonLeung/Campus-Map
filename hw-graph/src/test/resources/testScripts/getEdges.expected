# Tests the getChildren function by adding edges, printing the edges and then
# adding another edge and printing it to see if new edge is printed

# Create graph
created graph getEdge

# Add Nodes
added node a to getEdge
added node b to getEdge

# Add edge
added edge AB from a to b in getEdge

# Print children
the children of a in getEdge are: b(AB)

# Add another edge
added edge AA from a to a in getEdge

# Print children
the children of a in getEdge are: a(AA) b(AB)